lib/trino-filesystem-azure/src/test/java/io/trino/filesystem/azure/AbstractTestAzureFileSystem.java

```diff
@@ -182,7 +182,7 @@ protected final boolean isHierarchical()
     @Override
     protected boolean supportsPreSignedUri()
     {
-        return azureAuth.supportsPreSignedUri();
+        return true;
```

Comment:
when do you set false?

Reply:
Actually it's not needed anymore I think. Let me check

Reply:
it's false in the parent class. this is true for s3/gcs/abfs but not for hdfs or local file system